notebook/notebook.html

```diff
@@ -338,6 +338,7 @@
 {{super()}}
 <script type="text/javascript">
     sys_info = {{sys_info|safe}};
+    iframe_source = "{{iframe_source|default('http://localhost')}}";
```

Comment:
not even `var`?

Reply:
I wanted it to be consistent with the default var above 🤷  I'll try adding `var` to both, hopefully it won't break things

Reply:
Added and tested, works 👍